worksheet_B/README.md

```diff
@@ -0,0 +1,20 @@
+    List = Premade list of random words
+    Import Random
+    Import SequenceMatcher
+    randomWord = Random selects 1 word randomly from list
+    Tries = 0
+    While Tries < 4
+    Guess = Input
```

Comment:
For readability you should indent the code inside while loops, if statements etc. Note that to preserve indentation in a markdown file, pseudocode should be wrapped in ``` quotes.